src/utils/utils.py

```diff
@@ -188,7 +188,7 @@ def split_data(data, ratio, shuffle=1):
         splits[1].append(col[split_pt:])
     return tuple(splits[0]), tuple(splits[1])
     
-def unbind_predictions(self, preds: torch.Tensor) -> Iterable[np.ndarray]:
+def unbind_predictions(preds: torch.Tensor) -> Iterable[np.ndarray]:
```

Comment:
What's going on here? This method does get called in a few places, so this change should break the repo...

Reply:
It was previously a task method (that didn't need to be) and is pulled out, so this deletes `self`, but idk why it wasn't raising issues earlier.

Reply:
But isn't it called with [something].unbind_predictions() elsewhere? Won't that break?

Reply:
@pruksmhc Can you confirm that all the calls to this are non-broken?

Reply:
...confirmed?